src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -847,6 +847,12 @@ def load_arguments(self, _):
                        help='The nodepool id.', validator=validate_nodepool_id)
             c.argument('aks_custom_headers')
 
+    for scope in ['aks nodepool snapshot update']:
+        with self.argument_context(scope) as c:
+            c.argument('snapshot_name', options_list=[
+                       '--name', '-n'], required=True, help='The nodepool snapshot name.', validator=validate_snapshot_name)
+            c.argument('tags', tags_type, required=True, help='The tags to set to the snapshot.')
```

Comment:
```suggestion
    with self.argument_context('aks nodepool snapshot update') as c:
        c.argument('snapshot_name', options_list=['--name', '-n'], required=True, help='The nodepool snapshot name.', validator=validate_snapshot_name)
        c.argument('tags', tags_type, required=True, help='The tags to set to the snapshot.')
```

Reply:
> required=True

If the default value of argument is not set in the custom method, then the `required=True` is the default setting, it is no need to explicitly specify it again